runtime/permissions.rs

```diff
@@ -1044,7 +1058,7 @@ impl UnaryPermission<FfiDescriptor> {
     } else {
       let state = self.query(None);
       if state == PermissionState::Prompt {
-        if permission_prompt("ffi access", self.name) {
+        if self.prompt && permission_prompt("ffi access", self.name) {
```

Comment:
I just stumbled on this issue in my deno project. Just a minor nitpick. Given that this code self.prompt && permission_prompt is repeated in several places would it not make sense to just create a new function with it and call it?

Like

```rust
fn needsPrompt(self, msg: str) {
    return self.prompt && permission_prompt(msg, self.name);
}
```